proto/fungible/genesis.proto

```diff
@@ -2,15 +2,13 @@ syntax = "proto3";
 package zetachain.zetacore.fungible;
 
 import "fungible/foreign_coins.proto";
-import "fungible/params.proto";
 import "fungible/system_contract.proto";
 import "gogoproto/gogo.proto";
 
 option go_package = "github.com/zeta-chain/zetacore/x/fungible/types";
 
 // GenesisState defines the fungible module's genesis state.
 message GenesisState {
-  Params params = 1 [(gogoproto.nullable) = false];
-  repeated ForeignCoins foreignCoinsList = 2 [(gogoproto.nullable) = false];
-  SystemContract systemContract = 3;
+  repeated ForeignCoins foreignCoinsList = 1 [(gogoproto.nullable) = false];
```

Comment:
I think we should keep existing field indexes `2` and `3` untouched. Changing field indexes will probably break backward compatibility in `protobuf` context (for instance, the queries to genesis state). @kingpinXD  correct me if I'm wrong.